src/Parser/BalanceDruid/Modules/Spells/UnempoweredLs.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+class UnempoweredLS extends Module {
```

Comment:
I'd use the full spell name here to make the name a bit easier to understand, but it's up to you.